ergo/distributions/mixture.py

```diff
@@ -129,18 +139,39 @@ def from_conditions(
         initial_dist: Optional[M] = None,
         num_components: Optional[int] = None,
         verbose=False,
+        true_min=0,
+        true_max=1,
```

Comment:
Kind of feels like this function has too many parameters now. Out of scope for this PR but we should refactor somehow.